compiler/z/codegen/OMRCodeGenerator.hpp

```diff
@@ -956,10 +956,6 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGenerator
    // Snippet Data functions
    void addDataConstantSnippet(TR::S390ConstantDataSnippet * snippet);
 
-   // Identify the Inst selection phase
-   bool getDoingInstructionSelection() { return _cgFlags.testAny(S390CG_doingInstructionSelection); }
-   void setDoingInstructionSelection(bool b) { _cgFlags.set(S390CG_doingInstructionSelection, b); }
-
```

Comment:
Could you remove the flag `S390CG_doingInstructionSelection` in `compiler/z/codegen/OMRCodeGenerator.hpp` as well?  The usual way to do this is to just comment out the name with `// Available`.  There is an example of that already in that enum that you can just mimic.